src/oidcc_cowboy_callback.erl

```diff
@@ -222,3 +234,11 @@ retrieve_userinfo(Token, ProviderId, ClientId, ClientSecret, true) ->
 %% @private
 terminate(_Reason, _Req, _State) ->
     ok.
+
+normalize_scope(Scope) when is_binary(Scope) ->
```

Comment:
This one could go to `oidcc_scope`, but I'm not sure it is worth another PR, it is up to you.

Reply:
A PR for that would make sense. I have to do the same fix in the plug library i think.

Reply:
Turns out it is not needed, since the type is `scopes()` and it is normalized later